src/layout/Footer.scss

```diff
@@ -0,0 +1,10 @@
+footer {
+    // background-color: variables.$primary;
```

Comment:
Remove comments. I think we need to deine one code style. For me it will be better to use sx for styles in MUI, thats will remove all scss files in project and make it cleaner

Reply:
Move this styles to Footer.tsx in sx of Box and remove this file